CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Psammead Changelog
 
-| Version | Description |
-|---------|-------------|
-| 1.1.0   | [PR#336](https://github.com/BBC/psammead/pull/336) Add RTL Storybook add-on for testing |
-| 0.1.1   | [PR#354](https://github.com/BBC-News/psammead/pull/354) Update README and add initial changelog. |
-
+| Version | Description                                                                                           |
```

Comment:
```suggestion

<!-- prettier-ignore -->
| Version | Description                                                                                           |
```

If you put prettier ignore before `| Version | Description   |` it will stop the auto linting which some in the team complain about